src/AppModel/NetDaemon.AppModel/Internal/Config/ConfigurationBinder.cs

```diff
@@ -0,0 +1,1118 @@
+// This is a modified version of the .net 8 source code for ConfigurationBinder.cs
+// https://github.com/dotnet/runtime/blob/v8.0.7/src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.class
```

Comment:
Link doe snot seem to work

```suggestion
// https://github.com/dotnet/runtime/blob/2aade6beb02ea367fd97c4070a4198802fe61c03/src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs
```